src/app/content/practiceQuestions/components/QuestionNavigation/index.tsx

```diff
@@ -36,50 +36,65 @@ const QuestionNavigation = ({ question, selectedAnswer, ...props }: QuestionNavi
     {showSkipAndSubmit && <React.Fragment>
       <FormattedMessage id='i18n:practice-questions:popup:navigation:skip'>
         {(msg: string) => (
-          <PQInput
-            type='button'
-            value={msg}
+          <PQButton
+            size='large'
             withoutBg={true}
-            onClick={() => {
+            onClick={(e: React.MouseEvent) => {
+              e.preventDefault();
               dispatch(setAnswer({ answer: null, questionId: question.uid }));
               dispatch(nextQuestion());
             }}
-          />
+          >
+            {msg}
+          </PQButton>
         )}
       </FormattedMessage>
       <FormattedMessage id='i18n:practice-questions:popup:navigation:submit'>
         {(msg: string) => (
-          <PQInput
-            type='submit'
-            value={msg}
-            default={true}
+          <PQButton
+            variant='primary'
+            size='large'
             disabled={selectedAnswer ? false : true}
+            component={<input type='submit' value={msg} />}
```

Comment:
where is the onSubmit handler for this form?

Reply:
it is handled by `<form onSubmit={...} ...>` in `Question.tsx`